packages/core/src/external/carequality/ihe-gateway-v2/outbound/xca/process/dr-response.ts

```diff
@@ -90,35 +92,35 @@ async function parseDocumentReference({
     docUniqueId: documentResponse.DocumentUniqueId.toString(),
     metriportId: metriportId,
     fileLocation: bucket,
-    homeCommunityId: documentResponse.HomeCommunityId,
+    homeCommunityId: outboundRequest.gateway.homeCommunityId,
```

Comment:
fix for Redox DRs not being sent in prod. 
Now we are going to use homeCommunityId we specify instead of what they provide back